### PR TITLE
Feature/visit

### PR DESCRIPTION
**Pull request by KostetskyiYaroslav**
-implement feature to add visit as new visit to the doctor
-add possibility to view visits of patient on his page
image
image

### CODE REVIEW

File: MedicalDocManagment.WebUI/app/components/childrens-card/visit/list/list-visits.component.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+                        </div>
+                        <div>
+                            <b>Заключення:</b> {{visit.summary}} <br />
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div *ngIf="!visits || !visits.length">
+                <div>
+                    Дитини ще ні разу не була на примомі у лікарів.
```
Comment (AndriiKatsubo) [inline MedicalDocManagment.WebUI/app/components/childrens-card/visit/list/list-visits.component.html]: примомі

**Review by AndriiKatsubo — DISMISSED** [review]
Grammar mistake fixing was requested.

**Comment by AndriiKatsubo** [comment]
It's hard to add date from Firefox. Make date input for Firefox users more comfortable.
default

**Review by AndriiKatsubo — DISMISSED** [review]
Changes according to comment were requested.

**Comment by KostetskyiYaroslav** [comment]
image
image

File: MedicalDocManagment.WebUI/app/components/childrens-card/visit/list/list-visits.component.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+                <div *ngFor="let visit of visits" class="panel panel-info">
+                    <div class="panel-heading">
+                        <div>
+                            <b>Дата: </b>
+                            {{visit.date | amDateFormat: 'DD.MM.YYYY'}} <br />
+                        </div>
+                    </div>
+                    <div class="panel-body">
+                        <div>
+                            <b>Лікарь ( {{visit.doctor.position.name}} ):</b>
```
Comment (AndriiKatsubo) [inline MedicalDocManagment.WebUI/app/components/childrens-card/visit/list/list-visits.component.html:20]: Лікар

**Review by AndriiKatsubo — CHANGES_REQUESTED** [review]
Grammar mistake fixing was requested.

**Comment by AndriiKatsubo** [comment]
Menu item "Додати візит" doesn't open feature's window from children cards list.

**Review by AndriiKatsubo — CHANGES_REQUESTED** [review]
Changes according to comment message were requested.

**Pull request closed without merging**